server/routes/api.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const router = express.Router();
+const dexReq = require("./dexcom");
+const fitReq = require("./google");
+
+router.get('/egvs', (req, res) => {
```

Comment:
Add the `isAuthenticated` middleware here so that only authenticated users can reach this endpoint.  You should also add an additional middleware that validates that the user has Google and Dexcom auth tokens set